docs/resources/backup_schedule.md

```diff
@@ -462,6 +461,513 @@ This array is replaced during a strategic merge patch.
+  backup_scope = "FULL_CLUSTER"
+
+  selector{
+     excluded_names = [
+        "cluster1",
+        "cluster2"
+     ]
+  }
+  
+  spec {
+    schedule {
+      rate = "0 12 * * 1"
+    }
+
+    template {
+      backup_ttl          = "2592000s"
+      excluded_namespaces = [
+        "app-01",
+        "app-02",
+        "app-03",
+        "app-04"
+      ]
+      excluded_resources = [
+        "secrets",
+        "configmaps"
+      ]
+
+      storage_location = "TARGET_LOCATION_NAME"
+    }
+  }
+}
+```
+
+# Selected Namespaces Hourly Cluster Group Backup Schedule
+
+## Example Usage
+
+```terraform
+resource "tanzu-mission-control_backup_schedule" "sample-full" {
+  name                    = "namespaces-hourly"
+  scope {
+    scope {
+    cluster_group {
+      cluster_group_name            = "CG_NAME"
+    }
+  }
+
+  backup_scope                   = "SET_NAMESPACES"
+
+  selector{
+     excluded_names = [
+        "cluster1",
+        "cluster2"
+     ]
+  }
+  
+  spec {
+    schedule {
+      rate = "30 * * * *"
+    }
+
+    template {
+      included_namespaces = [
+        "app-01",
+        "app-02",
+        "app-03",
+        "app-04"
+      ]
+
+      excluded_resources = [
+        "secrets",
+        "configmaps"
+      ]
+
+      backup_ttl                = "86400s"
+      include_cluster_resources = true
+      storage_location          = "TARGET_LOCATION_NAME"
+
+      hooks {
+        resource {
+          name = "sample-config"
+
+          pre_hook {
+            exec {
+              command   = ["echo 'hello'"]
+              container = "workload"
+              on_error  = "CONTINUE"
+              timeout   = "10s"
+            }
+          }
+
+          pre_hook {
+            exec {
+              command   = ["echo 'hello'"]
+              container = "db"
+              on_error  = "CONTINUE"
+              timeout   = "30s"
+            }
+          }
+
+          post_hook {
+            exec {
+              command   = ["echo 'goodbye'"]
+              container = "db"
+              on_error  = "CONTINUE"
+              timeout   = "60s"
+            }
+          }
+
+          post_hook {
+            exec {
+              command   = ["echo 'goodbye'"]
+              container = "workload"
+              on_error  = "FAIL"
+              timeout   = "20s"
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+# Resources Selection By Label Selector Cluster Group Backup Schedule
+
+## Example Usage
+
+```terraform
+resource "tanzu-mission-control_backup_schedule" "sample-full" {
+  name                    = "label-based-no-storage"
+   scope {
+    cluster_group {
+      cluster_group_name            = "CG_NAME"
+    }
+  }
+
+  backup_scope                   = "LABEL_SELECTOR"
+  selector{
+     excluded_names = [
+        "cluster1",
+        "cluster2"
+     ]
+  }
+  spec {
+    schedule {
+      rate = "0 12 * * *"
+    }
+
+    template {
+      default_volumes_to_fs_backup = false
```

Comment:
nit: Use the terraform formatter to format examples

Reply:
Done